### PR TITLE
INTMDB-493: Serverless Private Endpoint Connection String Example Fix

### PR DESCRIPTION
**Pull request by martinstibbe**
## Description

Serverless Private Endpoint Connection String Example Fix

Link to any related issue(s):

## Type of change:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

File: mongodbatlas/resource_mongodbatlas_serverless_instance.go [hunk 1]
```diff
@@ -100,6 +100,14 @@ func returnServerlessInstanceSchema() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Computed: true,
 		},
+		"connection_strings_private_srv": {
+			Type:     schema.TypeList,
```
Comment (andreaangiolillo) [inline mongodbatlas/resource_mongodbatlas_serverless_instance.go:104]: [q] isn't this a string?

Comment (andreaangiolillo) [reply mongodbatlas/resource_mongodbatlas_serverless_instance.go:104]: oh this one is `privateEndpoint`, can we add `privateEndpoint` to the name to make it more clear?

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Review by Zuhairahmed — APPROVED** [review]
LGTM!

File: mongodbatlas/resource_mongodbatlas_serverless_instance.go (hunk 1 above)
Comment (martinstibbe) [reply mongodbatlas/resource_mongodbatlas_serverless_instance.go:104]: Renamed to connection_strings_private_endpoint_srv

**Pull request merged**